config/index.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   rDeployDirectory: '.rdeploy',
+  verifyEmail: true,
```

Comment:
Should we add an env var for this?

Reply:
I think env variables are for things that aren't configurable, like no sane defaults. For example, a database_url has no sane default. verifyEmail has a sane default though, and we don't want to overbloat `.env`